internal/git/git.go

```diff
@@ -376,10 +383,45 @@ func (g *Git) CommitAndPush(openAPIDocVersion, speakeasyVersion, doc string, act
 
 	var commitMessage string
 	if action == environment.ActionRunWorkflow {
-		commitMessage = fmt.Sprintf("ci: regenerated with OpenAPI Doc %s, Speakeasy CLI %s", openAPIDocVersion, speakeasyVersion)
 		if sourcesOnly {
 			commitMessage = fmt.Sprintf("ci: regenerated with Speakeasy CLI %s", speakeasyVersion)
+		} else {
+			// Do not add commitInfo to commit message if all values of releaseInfo are zero values
+			// Gate the sdk changelog release behind an env variable
+			if !isZeroReleasesInfo(releaseInfo) && os.Getenv("SDK_CHANGELOG_JULY_2025") == "true" && commitHeadings != nil && commitMessages != nil {
+				lang := ""
+				// count number of keys in releaseInfo.Languages
+				numberOfLanguagesGenerated := len(releaseInfo.LanguagesGenerated)
+				commitHeading := ""
+				newCommitMessage := ""
+				if numberOfLanguagesGenerated == 1 {
+					for key := range releaseInfo.Languages {
+						lang = key
+						break
+					}
+				} else {
+					// select a language from the released languages
+					for key := range releaseInfo.Languages {
+						lang = key
+						break
+					}
+				}
+				if numberOfLanguagesGenerated == 1 {
+					commitHeading = commitHeadings[lang]
```

Comment:
Should concatentate multiple here

```
ci: v1.2 @vercel/sdk
```

```
ci: Typescript v1.2 @vercel/sdk, C# ..., Java ...
```


but this is not useful: 

sdk.foo.bar() added
SDK.Foo.Bar() added